### DISCUSSION TITLE
How to add variant option on Dawn?

### DISCUSSION THREAD

**Discussion by annaoutlook** [Shopify/dawn#3152] [General]
There are some default variant options on Dawn Theme. But customers sometimes requires to add specific variant option to that.
How to solve this problem?

**Comment by top-comengineer** [comment]
Hi. Here is my answer.
Steps:
1. From the Shopify app, go to Products > All products.
2. Tap the product that you want to edit.
3. In the Variants section, tap XX variants.
4. Tap + .
5. Enter the option values in the existing options fields.
6. Optional: To add a product image to your new variant, click + , and then select a variant image. ...
7. Tap ✓.

**Reply by annaoutlook** [reply]
Wonderful, please. Thank you for your good answer.

**Comment by annaoutlook** [comment]
Wonderful, please. Thank you for your good answer.
